app/jobs/multi_region_kms_migration/profile_migration_job.rb

```diff
@@ -0,0 +1,58 @@
+module MultiRegionKmsMigration
+  class ProfileMigrationJob < ApplicationJob
+    MAXIMUM_ERROR_TOLERANCE = 10
+
+    include ::NewRelic::Agent::MethodTracer
+
+    def perform(statement_timeout: 120, profile_count: 1000)
+      error_count = 0
```

Comment:
can we add a feature flag check that early aborts? just in case we need to turn the job off for some reason?

Reply:
Yeah, I can do that

Reply:
just added a bit above this line